src/main/java/teammates/ui/controller/AdminSessionsPageAction.java

```diff
@@ -194,29 +168,28 @@ private ActionResult createAdminSessionPageResult(List<FeedbackSessionAttributes
         this.totalWaitToOpenStatusSessions = getTotalNumOfWaitToOpenStatusSession(allOpenFeedbackSessionsList);
 
         for (FeedbackSessionAttributes fs : allOpenFeedbackSessionsList) {
-
             List<InstructorAttributes> instructors = logic.getInstructorsForCourse(fs.getCourseId());
 
             if (instructors.isEmpty()) {
                 putIntoUnknownList(map, fs);
-            } else {
-                AccountAttributes account = getRegisteredInstructorAccountFromInstructors(instructors);
-
-                if (account == null) {
-                    putIntoUnknownList(map, fs);
-                    continue;
-                }
+                continue;
+            }
 
-                if (map.get(account.institute) == null) {
-                    List<FeedbackSessionAttributes> newList = new ArrayList<>();
-                    newList.add(fs);
-                    map.put(account.institute, newList);
-                } else {
-                    map.get(account.institute).add(fs);
-                }
+            AccountAttributes account = getRegisteredInstructorAccountFromInstructors(instructors);
+            if (account == null) {
+                putIntoUnknownList(map, fs);
+                continue;
+            }
 
+            if (map.get(account.institute) == null) {
+                List<FeedbackSessionAttributes> newList = new ArrayList<>();
+                newList.add(fs);
+                map.put(account.institute, newList);
```

Comment:
For future work (or you can do it now if you're up for it) - use an [`ArrayListMultimap`](https://google.github.io/guava/releases/22.0/api/docs/com/google/common/collect/ArrayListMultimap.html) instead of a map of lists for much cleaner code. I did similar work before in `BackDoorLogic#persistDataBundle`; you can check it out [here](https://github.com/TEAMMATES/teammates/pull/7770/commits/29581121c94e68fa9a92eb10e55f495bc9063894).

Reply:
I'll do another PR for this so that we don't have to block #8732 while I do the refactoring!